lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -336,15 +338,26 @@ ParseResult FIRParser::parseToken(FIRToken::Kind expectedToken,
   return emitError(message);
 }
 
-/// Parse a list of elements, terminated with an arbitrary token.
+/// Parse a comma-separated list of zero or more elements, terminated with an
+/// arbitrary token.
 ParseResult
 FIRParser::parseListUntil(FIRToken::Kind rightToken,
                           const std::function<ParseResult()> &parseElement) {
+  if (consumeIf(rightToken)) {
+    return success();
+  }
```

Comment:
nit: please remove braces https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
```suggestion
  if (consumeIf(rightToken))
    return success();
```